models/pancakeswap/bnb/pancakeswap_bnb_nft_market_v1_event.sql

```diff
@@ -0,0 +1,89 @@
+        evt_block_number AS block_number
+    FROM {{source('pancakeswap_v2_bnb','ERC721NFTMarketV1_evt_Trade')}}
+    {% if is_incremental() %}
+    WHERE evt_block_time >= date_trunc("day", now() - interval '1 week')
+    {% endif %}
+)
+    SELECT 
+        events.blockchain,
+        events.project,
+        events.version,
+        events.block_time,
+        date_trunc('day', events.block_time) AS block_date,
+        events.token_id,
+        bnb_nft_tokens.name collection,
+        events.amount_raw/POWER(10, bnb_bep20_tokens.decimals)*prices.price AS amount_usd,
+        events.token_standard,
+        CASE WHEN agg.name IS NOT NULL THEN 'Bundle Trade' ELSE 'Single Item Trade' END AS trade_type,
+        CAST(events.number_of_items AS DECIMAL(38,0)) AS number_of_items,
+        events.trade_category,
+        'Trade' AS evt_type,
+        events.seller,
+        events.buyer,
+        events.amount_raw/POWER(10, bnb_bep20_tokens.decimals) AS amount_original,
+        CAST(events.amount_raw AS DECIMAL(38,0)) AS amount_raw,
+        COALESCE(events.currency_symbol, bnb_bep20_tokens.symbol) AS currency_symbol,
+        events.currency_contract,
+        events.nft_contract_address,
+        events.project_contract_address,
+        agg.name AS aggregator_name,
+        CASE WHEN agg.name IS NOT NULL THEN agg.contract_address END AS aggregator_address,
+        events.tx_hash,
+        events.block_number,
+        bt.from AS tx_from,
+        bt.to AS tx_to,
+        CAST(0 AS DOUBLE) AS platform_fee_amount_raw,
+        0 AS platform_fee_amount,
+        0 AS platform_fee_amount_usd,
+        CAST(0 AS DOUBLE) AS platform_fee_percentage,
+        CAST(0 AS DOUBLE) AS royalty_fee_amount_raw,
+        0 AS royalty_fee_amount,
+        0 AS royalty_fee_amount_usd,
+        CAST(0 AS DOUBLE) AS royalty_fee_percentage,
+        0 AS royalty_fee_receive_address,
+        0 AS royalty_fee_currency_symbol,
+        events.blockchain || events.project || events.version || events.tx_hash || events.seller  || events.buyer || events.nft_contract_address || events.token_id AS unique_trade_id
+
+    FROM events
+    LEFT JOIN {{ ref('nft_aggregators') }} agg ON events.buyer=agg.contract_address AND agg.blockchain='bnb'
+    LEFT JOIN {{ ref('tokens_erc20') }} bnb_bep20_tokens ON bnb_bep20_tokens.contract_address=events.currency_contract AND bnb_bep20_tokens.blockchain='bnb'
+    LEFT JOIN {{ ref('tokens_nft') }} bnb_nft_tokens ON bnb_nft_tokens.contract_address=events.currency_contract AND bnb_nft_tokens.blockchain='bnb'
+    LEFT JOIN {{ source('prices', 'usd') }} prices ON prices.minute=date_trunc('minute', events.block_time)
+    AND (prices.contract_address=events.currency_contract AND prices.blockchain=events.blockchain)
+        {% if is_incremental() %}
+        AND prices.minute >= date_trunc("day", now() - interval '1 week')
+        {% endif %}
+    LEFT JOIN {{ source('bnb','transactions') }} bt ON bt.hash=events.tx_hash
```

Comment:
This can be made an inner join for better performance
```suggestion
    INNER JOIN {{ source('bnb','transactions') }} bt ON bt.hash=events.tx_hash
```